Modules/IO/RAW/include/itkRawImageIO.hxx

```diff
@@ -21,8 +21,27 @@
 #include "itkRawImageIO.h"
 #include "itkIntTypes.h"
 
+
 namespace itk
 {
+
+/** Utility function for writing RAW bytes */
+extern void
+WriteRawBytesAfterSwapping( itk::ImageIOBase::IOComponentType componentType,
```

Comment:
Isn't this identical re-declaration from .h file?

Reply:
Removed from .h file.  I put these extern declarations in the .hxx file as they are not intended to be used outside of this class.